.gitignore

```diff
@@ -1,4 +1,5 @@
 *.a
+!*/chirp/*.a
```

Comment:
I had already started on another branch X) Could you add the .vscode folder, it has nothing to do i the files pushed